### DISCUSSION TITLE
What is the best way to conditionally disable listening for messages using RabbitSubscribe?

### DISCUSSION THREAD

**Discussion by synic** [golevelup/nestjs#577] [General]
We are running our admin panel and customer facing website from the same codebase. Different webservers are set up to serve up different parts of the same codebase. 

Is there a way to conditionally disable the connection that `RabbitSubscribe` creates? We have a shared `JobsModule` that does the setup of the golevelup library. We've tried just conditionally not importing it, but then any references to `AmqpConnection` cannot be resolved

**Comment by WonderPanda** [comment]
Did you get a chance to try out [Conditional Handler Registration](https://github.com/golevelup/nestjs/tree/master/packages/rabbitmq#conditional-handler-registration)?
